algorithms/src/polycommit/sonic_pc/mod.rs

```diff
@@ -246,7 +251,7 @@ impl<E: PairingEngine, S: AlgebraicSponge<E::Fq, 2>> SonicKZG10<E, S> {
                 Ok((LabeledCommitment::new(label.to_string(), comm, degree_bound), rand))
             });
         }
-        let results: Vec<Result<_, PCError>> = pool.execute_all();
+        let results: Vec<Result<_, anyhow::Error>> = pool.execute_all();
```

Comment:
anyhow's Result is syntactic sugar precisely for this. The same likely applies to other places.
```suggestion
        let results: Vec<Result<_>> = pool.execute_all();
```